src/libtools/spline.cc

```diff
@@ -34,7 +34,9 @@ int Spline::GenerateGrid(double min, double max, double h) {
   }
   _r[i] = max;
   _f.resize(_r.size());
```

Comment:
is there a unit test for this

Reply:
because it changes behaviour, I am not saying it is wrong, it is probably better this way.

Reply:
I will probably have to create one, I know I have made a lot of changes that may not be thoughtfully tested, I wandering when this first gets released it should probably be released as an alpha, at least until we can get our code coverage up.